### PR TITLE
Category page added

### PR DESCRIPTION
**Pull request by surbhi84**
This PR consists of the following changes:

- Added category page and rules page.
- Made those pages responsive.
- Added navlinks.

### CODE REVIEW

File: src/pages/category/Category.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { CategoryCard } from "components";
+import "./category.css";
+const categoryData = [
+  {
```
Comment (ishitakeshawani) [inline src/pages/category/Category.tsx:4]: Suggestion: You can do API call here and get categoryData from it.

**Review by ishitakeshawani — APPROVED** [review]
Added a suggestion.
UI looks good.
I liked that you put comments in code. It is helpful.

**Review by rohanmathur91 — APPROVED** [review]
- Clean code
- Good use of `NavLink`
- Good that you have used typescript.

File: src/pages/category/Category.tsx (hunk 1 above)
Comment (surbhi84) [reply src/pages/category/Category.tsx:4]: yes I have yet to implement the backend and api calls.

**Pull request merged**